crates/cargo-contract/src/cmd/info.rs

```diff
@@ -50,6 +50,9 @@ pub struct InfoCommand {
         default_value = "ws://localhost:9944"
     )]
     url: url::Url,
+    /// Export the call output as JSON.
```

Comment:
```suggestion
    /// Export the instantiate output in JSON format.
```

This is a user-facing text, we should use the same description as other sub-commands do for`--output-json`.